CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - Changed: [[#293](https://github.com/ethereum/evmc/pull/293)]
   In C++ API `evmc::result::raw()` renamed to `evmc::result::release_raw()`.
+- Changed: [[#311](https://github.com/ethereum/evmc/pull/311)]
+  In `evmc_load_and_create()` the `error_code` is not optional (can be `NULL`).
```

Comment:
`not` or `now`?

Reply:
`now`. Why am I making the same mistake all the time?